examples/Demo/FluentUI.Demo.Client/Documentation/Components/TextArea/Examples/TextAreaAppearances.razor

```diff
@@ -0,0 +1,33 @@
+﻿<FluentStack Orientation="Orientation.Vertical">
+    <FluentStack Orientation="Orientation.Horizontal">
```

Comment:
You should use `HorizontalGap="12px" VerticalAlignment="VerticalAlignment.Bottom"`